packages/foam-vscode/static/graphs/default/graph.js

```diff
@@ -264,12 +300,12 @@ try {
   const vscode = acquireVsCodeApi();
 
   window.onload = () => {
-    initDataviz(vscode);
-    console.log("ready");
+    console.log("post webviewStyleRequest");
     vscode.postMessage({
-      type: "webviewDidLoad"
```

Comment:
mmm... here I would actually keep the `webviewDidLoad` event, which has a specific semantic (webview is ready, give me what I need, whether it's data, style, ..)

Reply:
Hm I see, I thought of it in another way. Load everything else first, the webview last

Reply:
So I tried to write 
```js
window.onload = () => {
  vscode.postMessage({
    type: "webviewStyleRequest",
  });
  initDataviz(vscode);
  console.log("ready");
  vscode.postMessage({
    type: "webviewDidLoad"
  });
};
```
And on the `didUpdateStyle`:
```js
case "didUpdateStyle":
        const style = message.payload;
        Actions.updateStyle(style);
```
And the graph does not update afterwards. I'm guessing the webview would need to be refreshed